### PR TITLE
Added support for w:fldSimple, including convenience for MailMerge

### PR DESCRIPTION
**Pull request by pauldegroot**
### Simple fields
This adds support for SimpleFields, which are found in Office Open XML documents as w:fldSimple. These are fields like `FILENAME` or `TITLE`.

You can use these within paragraphs just like in TextRuns, like this:
```typescript
const doc = new Document({
    sections: [{
        properties: {},
        children: [
            new Paragraph({
                children: [
                    new TextRun("This file is called "),
                    new SimpleField("FILENAME", "Example.docx"),
                    new TextRun("!"),
                ]
            })
        ],
    }],
});
```

### Convenience class: SimpleMailMergeField
A convenience class was also added to include the fields for a mail merge. This automatically creates a cached value that make the field look as if we created this field using Microsoft Word.

Basically, the code:

```typescript
new SimpleMailMergeField("Name")
```

is a shorthand for

```typescript
new SimpleField("MERGEFIELD Name", "«Name»")
```

### CODE REVIEW

**Comment by dolanmiu** [comment]
Nice work!

Could you create a file under `/demo` and write some documentation under `docs` about it?

**Comment by pauldegroot** [comment]
Sure! Hope this matches your expectations.

**Comment by dipakparmar** [comment]
@pauldegroot @dolanmiu do you know patching document with mail-merge fields is supported? I tried replacing `{{` and `}}` with `«` and `»` but it doesn't seem to work. 

https://github.com/dolanmiu/docx/blob/022b25cfcd33cd2ab9456f4b16cbc01ec3a1ed63/src/patcher/from-docx.ts#L130
